SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialGDKFunctionalTestsPrivate.h

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Logging/LogMacros.h"
+
+DECLARE_LOG_CATEGORY_EXTERN(LogSpatialGDKFunctionalTests, Log, All);
```

Comment:
We should also probably start to use this LogCategory instead of my old random TempLogs...

Reply:
Thought these were replaced already, but did a check and realised this was another log category that was added in a specific file. Replaced all with this module's LogSpatialGDKFunctionalTests log category + 1 left-over LogTemp